### PR TITLE
MicroProfile Metrics v5.1.0 TCK using RC1

### LINKED ISSUE #26033: Add MP Metrics 5.1 TCK
Currently, using RC1

### PR DESCRIPTION
**Pull request by Channyboy**
#build
fixes #26033

### CODE REVIEW

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.microprofile.metrics</groupId>
+    <artifactId>tck.runner</artifactId>
+    <version>5.0</version>
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/pom.xml]: 5.1?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/tck/pom.xml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <version>4.0.0</version>
+            <scope>provided</scope>
+        </dependency>
+        
+        <dependency>
+		    <groupId>io.openliberty.arquillian</groupId>
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/tck/pom.xml]: Indentation

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/tck/pom.xml [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+            <artifactId>arquillian-liberty-managed-jakarta</artifactId>
+            <version>${arquillian.liberty.managed.jakarta.version}</version>
+            <scope>test</scope>
+        </dependency>
+        
+		<dependency>
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat_tck/publish/tckRunner/tck/pom.xml]: Indentation

**Review by fmhwong — DISMISSED** [review]
LGTM with minor comments

**Comment by Channyboy** [comment]
#libby

**Review by fmhwong — APPROVED** [review]
LGTM

**Pull request merged**